README.md

```diff
@@ -0,0 +1,7 @@
+# 🥋 Mindset Dojo & Negotiation Mastery Program
+
+Start your journey today: build emotional wisdom, master authentic conversation, and unlock your full leadership potential at https://dojo.center/
+
+See the community at https://dojo.center/community
+
+Please respect the CC BY-SA 4.0 license when contributing.
```

Comment:
DRY - don't repeat yourself ... link directly to the license (in case it changes in the future)

Please respect the [LICENSE](./LICENSE.md)